games/openrct2/Portfile

```diff
@@ -4,9 +4,9 @@ PortSystem          1.0
 PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        OpenRCT2 OpenRCT2 0.1.2 v
+version             0.3.2
+github.setup        OpenRCT2 OpenRCT2 ${version} v
```

Comment:
If you just put ```0.3.2``` where ${version} is here, it will work correctly, and also set ${version} to ```0.3.2``` for you. So you don't need to add this complexity here to get the same result.